### PR TITLE
Archive repo

### PR DESCRIPTION
**Pull request by winkingturtle-vmw**
- [x] Read the [Contributing document](../blob/-/.github/CONTRIBUTING.md).

Summary
---------------
Archive repo

Backward Compatibility
---------------
Breaking Change? **No**

### CODE REVIEW

**Comment by MarcPaquette** [comment]
Is the documentation provided in the readme moved to the  other repo as well? Where is it?

**Comment by winkingturtle-vmw** [comment]
Yes. https://github.com/cloudfoundry/mapfs-release/blob/develop/docs/01-overview.md

File: README.md
Comment (MarcPaquette) [inline README.md:1]: I think if we could get a link to the documentation and the exact location of the source code relocation it should help folks trace it back. We should also have a link from the new repository to this old repository so folks can track down commit history if needed.

File: README.md [hunk 1]
```diff
@@ -1,63 +1,5 @@
...
-This repository is maintained by [Service
-Management](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/service-management.md)
-under `Volume Services` area.
-
-> \[!IMPORTANT\]
->
-> Content in this file is managed by the [CI task
-> `sync-readme`](https://github.com/cloudfoundry/wg-app-platform-runtime-ci/blob/c83c224ad06515ed52f51bdadf6075f56300ec93/shared/tasks/sync-readme/metadata.yml)
-> and is generated by CI following a convention.
+It's documentation has been relocated to the [docs section of the mapfs-release](https://github.com/cloudfoundry/mapfs-release/blob/develop/docs/01-overview.md).
```
Comment (ebroberson) [inline README.md:5]: It's => Its

**Review by ebroberson — APPROVED** [review]
LGTM

**Review by geofffranks — APPROVED** [review]

**Pull request merged**